packages/@aws-cdk/aws-synthetics/README.md

```diff
@@ -57,10 +58,10 @@ const log = require('SyntheticsLogger');
 const pageLoadBlueprint = async function () {
 
     // INSERT URL here
-    const URL = "https://api.example.com/user/books/topbook/";
+    const url = process.env.URL;
```

Comment:
If the same canary should perform the same work but on different stage APIs. I'll update this to be a bit more interesting

Reply:
dont use `process.env.URL` as it may create the impression that the `URL` env variable is set by the service, if you want to give an example than add it as a comment